lawnchair/src/app/lawnchair/ui/preferences/GeneralPreferences.kt

```diff
@@ -67,6 +68,22 @@ fun GeneralPreferences() {
                 )
             }
         }
+        PreferenceGroup(heading = stringResource(id = R.string.notification_dots)) {
+            val context = LocalContext.current
+            val enabled by remember { notificationDotsEnabled(context) }.collectAsState(initial = false)
+            val serviceEnabled = notificationServiceEnabled()
+            NotificationDotsPreference(enabled = enabled, serviceEnabled = serviceEnabled)
+            AnimatedVisibility(
+                visible = enabled && serviceEnabled,
+                enter = expandVertically() + fadeIn(),
+                exit = shrinkVertically() + fadeOut(),
+            ) {
+                SwitchPreference(
+                    adapter = prefs2.showNotificationCount.getAdapter(),
+                    label = stringResource(id = R.string.show_notification_count),
+                )
+            }
+        }
```

Comment:
Sorry, one more request.
This doesn't need to be animated because toggling the notification service and notification dots are done in the system's Settings app.